cirq-google/cirq_google/experimental/noise_models/calibration_to_noise_properties.py

```diff
@@ -0,0 +1,164 @@
+# Copyright 2021 The Cirq Developers
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Dict, Tuple, Type
+import numpy as np
+
+import cirq, cirq_google
+from cirq.devices import noise_utils
+from cirq_google.devices import google_noise_properties
```

Comment:
Do we need these imports if we are doing broader imports just above ?

Reply:
Resolved...ish. See comment on circular imports :/